CHANGES_NEXT_RELEASE

```diff
@@ -1,4 +1,4 @@
 [cygnus-ngsi][MySQLSink, PostgreSQLSink, PostgisSQLSink] Remove SQLBackend singleton, thus allowing two sinks of the same SQL-type in the same Cygnus instance
 [cygnus-ngsi][CKANSink] New datamodel for CKAN (dm-by-entity-id) implementing mapping: subservice -> org, entityId -> dataset, entityId -> resource (#1792)
 [cygnus-ngsi][PostgisSQLSink, PostgreSQLSink] Add dm-by-entity-database and dm-by-entity-database-schema.
-
+[cygnus-ngsi][PostgisSQLSink] Look for a "geo:something" attribute type on mapped context element of an event before generic aggregation
```

Comment:
```suggestion
[cygnus-ngsi][PostgisSQLSink] Look for a "geo:json" or "geo:point" attribute type on mapped context element of an event before generic aggregation
```

Not sure but looking to the code modifications it seems "geo:something" only have these two possiblities.